db_test.go

```diff
@@ -207,11 +207,12 @@ func TestDB_Merge_For_string(t *testing.T) {
 	db2, err := Open(
 		DefaultOptions,
 		WithDir(fileDir),
+		WithNodeNum(1011),
 		WithSegmentSize(1*100),
 	)
 
 	require.NoError(t, err)
-
```

Comment:
Seems this function is for testing the merge function, not for testing the Options function. Could you help add it in the rightful place?

Reply:
ok

Reply:
Resolved